client/src/components/entry/SignUpForm.js

```diff
@@ -0,0 +1,120 @@
+import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import SocialSignUp from './SocialSignUp';
+import AuthService from '../../service/AuthService';
+
+export default function SignUpForm() {
+  const [user, setUser] = useState({ username: '', password: '' });
+  const [isMatch, setMatch] = useState(true);
+  const [isEmpty, setEmpty] = useState(false);
+  const [confirmInput, setConfirmInput] = useState({ usernameConfirm: '', passwordConfirm: '' });
+  const [errMessage, setErrMessage] = useState('');
+
+  const history = useHistory();
+
+  const onChange = (event) => {
+    setUser({ ...user, [event.target.name]: event.target.value });
+  };
+
+  const onConfirmation = (event) => {
```

Comment:
because this function is being called when the input field is changed, I don't think the name `onConfirmation` fits the best. Maybe something like `setConfirmation`, `updateConfirmation`, `setConfirmationInput` or `updateConfirmationInput`?

Reply:
i agree, i had to think about this as well. adjustments made